priorityq.py

```diff
@@ -0,0 +1,38 @@
+#! /usr/bin/env python
+# Priority Queue Python file for Efrain, Mark and Henry
+import queue as q
+
+
+class Pq(object):
+    """Initiate priority queue"""
+
+    def __init__(self, num=3):
+        """Initiate pq with optional priority range, default is 3."""
+        for x in range(1, num+1):
+            setattr(self, 'que{}'.format(x), q.Queue())
+        self.num_queues = num
+
+    def insert(self, value, priority='3'):
+        """Insert an item into a pq, according to priority."""
+        try:
+            getattr(self, 'que{}'.format(priority)).enqueue(value)
+        except AttributeError:
+            getattr(self, 'que{}'.format(self.num_queues)).enqueue(value)
```

Comment:
when is this `AttributeError` raised?  Is it better to default to the lowest priority , or to inform the user that they've made an error by selecting a priority that doesn't exist.  

I'd argue the latter is better, since the former involves altering the user's data without their explicit consent.